artio-codecs/src/test/java/uk/co/real_logic/artio/fields/DecimalFloatToAndFromDoubleTest.java

```diff
@@ -31,40 +32,39 @@ public void shouldExtractNaNFromDecimalNumber()
     @Test
     public void shouldConvertFromDoubleAccurately()
     {
-        final StringBuilder builder = new StringBuilder();
```

Comment:
I don't think these decimal float changes look related to the indexer catchup, can you keep them out of the PR please.

Reply:
removed. In Russia this test fails :)